app/src/main/java/tech/pauly/findapet/shared/ActivityEvent.java

```diff
@@ -0,0 +1,63 @@
+package tech.pauly.findapet.shared;
+
+public class ActivityEvent extends BaseViewEvent {
+
+    public enum ActivityEventType {
+        START,
+        FINISH;
```

Comment:
Remove this, was planning on finish activities manually, but was overengineered, using the support action bar 
 up button default implementation instead.